modules/exploits/windows/http/hp_pcm_snac_update_certificates.rb

```diff
@@ -40,17 +40,14 @@ def initialize(info = {})
       'Arch'           => ARCH_JAVA,
       'DefaultOptions' =>
         {
-          'SHELL' => 'cmd.exe'
+          'SHELL' => 'cmd.exe',
```

Comment:
This default SHELL option is used for the other hp_* exploits, though it feels like there may be some blind copy/paste between http modules.